diff --git a/node_modules/react-native/Libraries/Lists/VirtualizedList.js b/node_modules/react-native/Libraries/Lists/VirtualizedList.js
index 258a12e..3b3f3d2 100644
--- a/node_modules/react-native/Libraries/Lists/VirtualizedList.js
+++ b/node_modules/react-native/Libraries/Lists/VirtualizedList.js
@@ -407,7 +407,9 @@ class VirtualizedList extends React.PureComponent<Props, State> {
       return;
     }
 
-    this._scrollRef.scrollTo(
+const scrollComponent = this._scrollRef?.getNode ? this._scrollRef.getNode() : this._scrollRef;
+
+    scrollComponent.scrollTo(
       horizontalOrDefault(this.props.horizontal)
         ? {x: offset, animated}
         : {y: offset, animated},
